website/content/docs/api-gateway/upgrade-specific-versions.mdx

```diff
@@ -9,11 +9,186 @@ description: >-
+
+### Requirements
+
+Ensure that the following requirements are met prior to upgrading:
+
+- Consul API Gateway should be running version v0.2.1 or lower.
+- You should have the ability to run `kubectl` CLI commands.
+- `kubectl` should be configured to point to the cluster containing the installation you are upgrading.
+- You should have the following permission rights on your Kubernetes cluster:
+  - `Gateway.read`
+  - `ReferencePolicy.create`
+- (Optional) The [jq](https://stedolan.github.io/jq/download/) command line processor for JSON can be installed, which will ease route retrieval during the upgrade process.
+
+### Procedure
+
+
+1. Verify the current version of the `consul-api-gateway-controller` `Deployment`:
+
+  ```shell-session
+  $ kubectl get deployment --namespace consul consul-api-gateway-controller --output=jsonpath= "{@.spec.template.spec.containers[?(@.name=='api-gateway-controller')].image}"
+  ```
+
+  You should receive a response similar to the following:
+
+  ```log
+  "hashicorp/consul-api-gateway:0.2.1"
+  ```
+
+1. Retrieve all gateways that have a secret in a different namespace. If you have installed the [`jq`](https://stedolan.github.io/jq/) utility, you can skip to [step 4](#jq-command-secrets). Otherwise, issue the following command to get all `Gateways` across all namespaces:
+
+  ```shell-session
+  $ kubectl get Gateway --output json --all-namespaces
+  ```
+
+
+  If you have any active `Gateways`, you will receive output similar to the following response. The output has been truncated to show only relevant fields:
+
+  ```yaml
+    apiVersion: v1
+    items:
+    - apiVersion: gateway.networking.k8s.io/v1alpha2
+      kind: HTTPRoute
+      metadata:
+        name: example-http-route,
+        namespace: example-namespace,
+        ...
+      spec:
+        parentRefs:
+        - group: gateway.networking.k8s.io
+          kind: Gateway
+          name: gateway
+          namespace: gw-ns
+        rules:
+        - backendRefs:
+          - group: ""
+            kind: Service
+            name: web-backend
+            namespace: gateway-namespace
+            ...
+          ...
+    - apiVersion: gateway.networking.k8s.io/v1alpha2
+      kind: TCPRoute
+      metadata:
+        name: example-tcp-route,
+        namespace: a-different-namespace,
+        ...
+      spec:
+        parentRefs:
+        - group: gateway.networking.k8s.io
+          kind: Gateway
+          name: gateway
+          namespace: gateway-namespace
+        rules:
+        - backendRefs:
+          - group: ""
+            kind: Service
+            name: web-backend
+            namespace: gateway-namespace
+            ...
+      ...
+  ```
```

Comment:
This is rendered incorrectly

<img width="993" alt="image" src="https://user-images.githubusercontent.com/7556325/174875907-076a5e76-9aee-4998-86ad-6b7c9fc845b8.png">

````suggestion
    If you have any active `Gateways`, you will receive output similar to the following response. The output has been truncated to show only relevant fields:

  ```yaml
    apiVersion: v1
    items:
    - apiVersion: gateway.networking.k8s.io/v1alpha2
      kind: HTTPRoute
      metadata:
        name: example-http-route,
        namespace: example-namespace,
        ...
      spec:
        parentRefs:
        - group: gateway.networking.k8s.io
          kind: Gateway
          name: gateway
          namespace: gw-ns
        rules:
        - backendRefs:
          - group: ""
            kind: Service
            name: web-backend
            namespace: gateway-namespace
            ...
          ...
    - apiVersion: gateway.networking.k8s.io/v1alpha2
      kind: TCPRoute
      metadata:
        name: example-tcp-route,
        namespace: a-different-namespace,
        ...
      spec:
        parentRefs:
        - group: gateway.networking.k8s.io
          kind: Gateway
          name: gateway
          namespace: gateway-namespace
        rules:
        - backendRefs:
          - group: ""
            kind: Service
            name: web-backend
            namespace: gateway-namespace
            ...
      ...
  ```
````